src/html/shared/header.html

```diff
@@ -0,0 +1,21 @@
+<nav class="c-header  is-scroll" style="background-color:white;">
+
+    <button class="c-header__icon  c-icon  c-icon--vsmall">{{sprite("menu")}}</button>
+
+    <img src="../images/logo-darkbg.png" alt="Shopiflight logo" class="c-header__logo">
```

Comment:
@xavierang we should get an SVG of the logo so it's retina friendly and scaleable 👍